.travis.yml

```diff
@@ -19,10 +19,11 @@ env:
     - NOCOLOR='\033[0m'
     - BOLD=$(tput bold)
 
-addons:
-  apt:
-    packages:
-    - MbedTLS
+before_script:
+  - julia -e 'using Pkg;
+      Pkg.add(PackageSpec(name="MbedTLS", version="0.6.6"));
+      Pkg.pin(PackageSpec(name="MbedTLS", version="0.6.6"));
+      Pkg.add([PackageSpec(name="BinaryBuilder", rev="master"), PackageSpec(name="BinaryProvider")])'
```

Comment:
Is the use of the master branch of `BinaryBuilder` here strictly necessary? `MbedTLS` worked before so there should be a working version METADATA. Depending on the master branch of an external repo kinda asks for having an unstable ci.

Also, why do you add `BinaryBuilder` twice? And why is there a transpose oeprator behind the command?

Reply:
If you think, the `BinaryBuilder` is not necessary, I take it out.
Concerning the "transpose operator". In line 24 there stands: "julia -e 'using..." and it is the start of the command which ends in line 26 with the "transpose operator".